### ISSUE TITLE
Issue with k8s.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/

### ISSUE THREAD

**Issue by lstday** [kubernetes/website#15870]
**This is a Bug Report**

**403 Forbidden**

No index.html in mounted volume
```
root@task-pv-pod:/# mount | grep nginx
/dev/mapper/rhel-root on /usr/share/nginx/html type xfs (rw,relatime,attr2,inode64,noquota)
root@task-pv-pod:/# ls /usr/share/nginx/html
root@task-pv-pod:/# 
root@task-pv-pod:/# exit
[root@xxx-TECH1P ~]# ls /mnt/data/
index.html
```

RHEL, Kubernetes 1.15.2, latest Kubespray, Docker 19.03.1, Flannel

Upd: localhost, Fedora 30, KinD, docker 19.03.1, same thing.

**Comment by lstday** [comment]
Minikube on localhost same thing :(

**Comment by sftim** [comment]
@lstday, can you summarize what you have to do to replicate this issue? 
I checked these steps and did not see any problem.

/triage needs-information
